### PR TITLE
Fix readme

### PR DESCRIPTION
**Pull request by bc118**
I added some text to workflows as Jenny suggested, as it didn't really have much

If I am missing any engines please let me know.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
-### MoSDeF Workflows
+### MoSDeF Workflows -- A collection of sample workflows using MoSDeF tools
 
-A collection of sample workflows using MoSDeF tools
+The Molecular Simulation Design Framework [(MoSDeF)] (https://mosdef.org) tools, supports a wide range of simulation engines, including `Cassandra <https://cassandra.nd.edu), [GPU Optimized Monte Carlo (GOMC)] (http://gomc.eng.wayne.edu), [GROMACS] (https://www.gromacs.org), [HOOMD-blue] (http://glotzerlab.engin.umich.edu/hoomd-blue/), and [Large-scale Atomic/Molecular Massively Parallel Simulator (LAMMPS)] (https://lammps.sandia.gov).  The **MoSDeF** tools allow simulation reproducibility across the various simulation engines, eliminating the need to be an expert user in all the engines to replicate, continue, or advance the existing research.  Additionally, the software can auto-generate many different systems, allowing large-scale screening of chemicals and materials using [Signac] (https://signac.io) to manage the simulations and data.
```
Comment (jennyfothergill) [inline README.md:3]:
```suggestion
The Molecular Simulation Design Framework [(MoSDeF)](https://mosdef.org) tools support a wide range of simulation engines, including [Cassandra](https://cassandra.nd.edu), [GPU Optimized Monte Carlo (GOMC)](http://gomc.eng.wayne.edu), [GROMACS](https://www.gromacs.org), [HOOMD-blue](http://glotzerlab.engin.umich.edu/hoomd-blue/), and [Large-scale Atomic/Molecular Massively Parallel Simulator (LAMMPS)](https://lammps.sandia.gov).  The **MoSDeF** tools allow simulation reproducibility across the various simulation engines, eliminating the need to be an expert user in all the engines to replicate, continue, or advance the existing research.  Additionally, the software can auto-generate many different systems, allowing large-scale screening of chemicals and materials using [Signac](https://signac.io) to manage the simulations and data.
```
Let's fix markdown link format

File: README.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
...
+
+The [MoSDeF] (https://mosdef.org) software is comprised the following major packages:
+- [mBuild] (https://mbuild.mosdef.org/en/stable/) -- A hierarchical, component based molecule builder
+- [foyer] (https://foyer.mosdef.org/en/stable/) -- A package for atom-typing as well as applying and disseminating forcefields
+- [GMSO] (https://gmso.mosdef.org/en/stable/) -- Flexible storage of chemical topology for molecular simulation
```
Comment (jennyfothergill) [inline README.md:8]:
```suggestion
The [MoSDeF](https://mosdef.org) software is comprised the following major packages:
- [mBuild](https://mbuild.mosdef.org/en/stable/) -- A hierarchical, component based molecule builder
- [foyer](https://foyer.mosdef.org/en/stable/) -- A package for atom-typing as well as applying and disseminating forcefields
- [GMSO](https://gmso.mosdef.org/en/stable/) -- Flexible storage of chemical topology for molecular simulation
```
more link fixes

**Review by jennyfothergill — APPROVED** [review]
LGTM

File: README.md [hunk 3]
```diff
@@ -1,3 +1,8 @@
-### MoSDeF Workflows
+### MoSDeF Workflows -- A collection of sample workflows using MoSDeF tools
 
-A collection of sample workflows using MoSDeF tools
+The Molecular Simulation Design Framework [(MoSDeF)](https://mosdef.org) tools support a wide range of simulation engines, including [Cassandra](https://cassandra.nd.edu), [GPU Optimized Monte Carlo (GOMC)](http://gomc.eng.wayne.edu), [GROMACS](https://www.gromacs.org), [HOOMD-blue](http://glotzerlab.engin.umich.edu/hoomd-blue/), and [Large-scale Atomic/Molecular Massively Parallel Simulator (LAMMPS)](https://lammps.sandia.gov).  The **MoSDeF** tools allow simulation reproducibility across the various simulation engines, eliminating the need to be an expert user in all the engines to replicate, continue, or advance the existing research.  Additionally, the software can auto-generate many different systems, allowing large-scale screening of chemicals and materials using [Signac](https://signac.io) to manage the simulations and data.
```
Comment (umesh-timalsina) [inline README.md:3]:
```suggestion
The Molecular Simulation Design Framework [(MoSDeF)](https://mosdef.org) tools support a wide range of simulation engines, including [Cassandra](https://cassandra.nd.edu), [GPU Optimized Monte Carlo (GOMC)](http://gomc.eng.wayne.edu), [GROMACS](https://www.gromacs.org), [HOOMD-blue](http://glotzerlab.engin.umich.edu/hoomd-blue/), and [Large-scale Atomic/Molecular Massively Parallel Simulator (LAMMPS)](https://lammps.sandia.gov).  **MoSDeF** tools facilitate simulation reproducibility across the various simulation engines, thus lowering the barrier of entry to replicate, continue, or advance the existing research.  Additionally, the framework supports plethora of diverse systems, allowing large-scale screening of chemicals and materials using [Signac](https://signac.io) to manage the simulations and data.
```

**Review by umesh-timalsina — COMMENTED** [review]
Minor rewording.

File: README.md [hunk 4]
```diff
@@ -1,3 +1,8 @@
-### MoSDeF Workflows
+### MoSDeF Workflows -- A collection of sample workflows using MoSDeF tools
 
-A collection of sample workflows using MoSDeF tools
+The Molecular Simulation Design Framework [(MoSDeF)](https://mosdef.org) tools support a wide range of simulation engines, including [Cassandra](https://cassandra.nd.edu), [GPU Optimized Monte Carlo (GOMC)](http://gomc.eng.wayne.edu), [GROMACS](https://www.gromacs.org), [HOOMD-blue](http://glotzerlab.engin.umich.edu/hoomd-blue/), and [Large-scale Atomic/Molecular Massively Parallel Simulator (LAMMPS)](https://lammps.sandia.gov).  **MoSDeF** tools facilitate simulation reproducibility across the various simulation engines, thus lowering the barrier of entry to replicate, continue, or advance the existing research.  Additionally, the framework supports plethora of diverse systems, allowing large-scale screening of chemicals and materials using [Signac](https://signac.io) to manage the simulations and data.
```
Comment (umesh-timalsina) [inline README.md:3]:
```suggestion
The Molecular Simulation Design Framework [(MoSDeF)](https://mosdef.org) tools support a wide range of simulation engines, including [Cassandra](https://cassandra.nd.edu), [GPU Optimized Monte Carlo (GOMC)](http://gomc.eng.wayne.edu), [GROMACS](https://www.gromacs.org), [HOOMD-blue](http://glotzerlab.engin.umich.edu/hoomd-blue/), and [Large-scale Atomic/Molecular Massively Parallel Simulator (LAMMPS)](https://lammps.sandia.gov).  **MoSDeF** tools facilitate simulation reproducibility across the various simulation engines, thus lowering the barrier of entry to replicate, continue, or advance existing research.  Additionally, the framework supports plethora of diverse systems, allowing large-scale screening of chemicals and materials using [Signac](https://signac.io) to manage the simulations and data.
```

**Pull request merged**